### DISCUSSION TITLE
MJML - How to get images to align left & right but still stack on top of eachother on mobile

### DISCUSSION THREAD

**Discussion by sumsho** [mjmlio/mjml#2825] [Q&A]
This what i have right now: 
https://mjml.io/try-it-live/DhI5ofFajP

I'm trying to get the Logo image to be in the left corner, with the button being on the very right corner. But then in mobile i'd still like them to stack on each other and be aligned in the center. Right now the logo image and button are too close to the middle. 

Any help appreciated, thanks!

**Comment by Cwasiniak** [comment]
try creating multiple columns, maybe try three. Set the middle column to a specific width that allows for space between your right column with text, and your left column with logo. then once the mobile screen is shown the space should be adequate. creating empty columns can create space when specific widths are set, making it easier to play around with spacing without having to worry about responsiveness between screen sizes.
